src/language-server/lox-validator.ts

```diff
@@ -37,6 +38,13 @@ export class LoxValidator {
         this.checkFunctionReturnTypeInternal(method.body, method.returnType, accept);
     }
 
+    // TODO: implement classes 
+    checkClassDeclaration(declaration: Class, accept: ValidationAcceptor): void {
+        accept('error', 'Classes are currently unsupported.', {
+            node: declaration
```

Comment:
Suggestion: We should use `property: 'name'` here, otherwise the diagnostic covers the whole class (which could be potentially quite large).